changelogs/fragments/negate-command-vyos.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - "'prefix' added to NetworkTemplate class, inorder to handle the negate operation for vyos config commands."
```

Comment:
Small comment: Can we make the changelog a bit generic since it facilitates platforms other than vyos as well?